docs/walkthrough.md

```diff
@@ -0,0 +1,421 @@
+    reason: ProvisionedSuccessfully
+    status: "True"
+    type: Ready
+```
+
+## Step 8 - Bind to the Instance
+
+Now that our `Instance` has been created, we can bind to it. To accomplish this,
+we will create a [`Binding`](../contrib/examples/walkthrough/ups-binding.yaml)
+resource.
+
+```console
+kubectl --context=service-catalog create -f contrib/examples/walkthrough/ups-binding.yaml
+```
+
+That command should output:
+
+```console
+binding "ups-binding" created
+```
+
+After the `Binding` resource is created, the service catalog controller will
+communicate with the appropriate broker server to initiate binding. Generally,
+this will cause the broker server to create and issue credentials that the
+service catalog controller will insert into a Kubernetes `Secret`. We can check
+the status of this process like so:
+
+```console
+kubectl --context=service-catalog get bindings -n test-ns ups-binding -o yaml
+```
+
+We should see something like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Binding
+metadata:
+  creationTimestamp: 2017-03-07T01:44:36Z
+  finalizers:
+  - kubernetes
+  name: ups-binding
+  namespace: test-ns
+  resourceVersion: "29"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/bindings/ups-binding
+  uid: 9eb2cdce-02d7-11e7-8edb-0242ac110005
+spec:
+  instanceRef:
+    name: ups-instance
+  osbGuid: b041db94-a5a0-41a2-87ae-1025ba760918
+  secretName: my-secret
+status:
+  conditions:
+  - message: Injected bind result
+    reason: InjectedBindResult
+    status: "True"
+    type: Ready
+```
+
+Notice that the status has a `Ready` condition set.  This means our binding is
+ready to use!  If we look at the `Secret`s in our `test-ns` namespace, we should
+see a new one:
+
+```console
+kubectl get secrets -n test-ns
+NAME                  TYPE                                  DATA      AGE
+default-token-3k61z   kubernetes.io/service-account-token   3         29m
+my-secret             Opaque                                2         1m
+```
+
+Notice that a new `Secret` named `my-secret` has been created.
+
+## Step 9 - Unbind from the Instance
+
+Now, let's unbind from the instance.  To do this, we simply *delete* the
+`Binding` resource that we previously created:
+
+```console
+kubectl --context=service-catalog delete -n test-ns bindings ups-binding
+```
+
+Checking the `Secret`s in the `test-ns` namespace, we should see that
+`my-secret` has also been deleted:
+
+```console
+kubectl get secrets -n test-ns
+NAME                  TYPE                                  DATA      AGE
+default-token-3k61z   kubernetes.io/service-account-token   3         30m
+```
+
+## Step 10 - Deprovision the Instance
+
+Now, we can deprovision the instance.  To do this, we simply *delete* the
+`Instance` resource that we previously created:
+
+```console
+kubectl --context=service-catalog delete -n test-ns instances ups-instance
+```
+
+### Delete the Broker
```

Comment:
+1, this needs to be another step. I am going to fix this in a follow-up, since this PR makes significant progress in the walk-through